util/hash_map.h

```diff
@@ -0,0 +1,57 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.  This
+//  source code is licensed under the BSD-style license found in the LICENSE
+//  file in the root directory of this source tree. An additional grant of
+//  patent rights can be found in the PATENTS file in the same directory.
+//
+
+#pragma once
+
+#include <algorithm>
+#include <utility>
+#include <vector>
+
+namespace rocksdb {
+
+template <typename K, typename V, size_t size = 128>
+class HashMap {
+  std::vector<std::vector<std::pair<K, V>>> table_;
```

Comment:
Is there a way to use `std::vector<autovector<std::pair<K, V>, 1>> table_;` instead?